students/JonSerpas/lesson01/assignment/inventory_management/electricAppliancesClass.py

```diff
@@ -0,0 +1,26 @@
+# Electric appliances class
+from inventoryClass import inventory
+
+
+class electricAppliances(inventory):
+
+    def __init__(self, productCode, description, marketPrice,
+                 rentalPrice, brand, voltage):
+        inventory.__init__(self, productCode, description, marketPrice,
+                           rentalPrice)
+
+        # Creates common instance variables from the parent class
+
+        self.brand = brand
+        self.voltage = voltage
+
+    def returnAsDictionary(self):
+        outputDict = {}
+        outputDict['productCode'] = self.productCode
```

Comment:
Since lines 19-23 are the same as Inventory (the parent), you can remove them and add the following to the end of the method:
 ```outputDict = super().returnAsDictionary()```